src/software/simulated_tests/simulated_er_force_sim_play_test_fixture.h

```diff
@@ -70,7 +70,7 @@ class SimulatedErForceSimPlayTestFixture : public SimulatedErForceSimTestFixture
     AIDrawFunction getDrawFunctions() override;
 
    private:
-    void updatePrimitives(const World& world,
+    void updatePrimitives(const World& friendly_world, const World& enemy_world,
```

Comment:
javadocs

Reply:
these are overridden so we don't include javadocs